pkg/apis/v2v/v1beta1/virtualmachineimport_types.go

```diff
@@ -163,6 +163,9 @@ const (
 
 	// IncompleteMappingRules represents the inability to prepare the mapping rules
 	IncompleteMappingRules ValidConditionReason = "IncompleteMappingRules"
+
+	// ValidationReportedWarnings represents the existence of warnings as a result of checking the mapping
```

Comment:
Would it be possible to rephrase `represents the existence of warnings as a result of checking the mapping`. It took me some time to understand what it means.

Reply:
What about "ValidationReportedWarnings represents the existence of warnings related to resource mapping validation"?

Reply:
Sounds good